ethnobiology-and-conservation.csl

```diff
@@ -1,116 +1,133 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0.1" demote-non-dropping-particle="sort-only" default-locale="en-US">
   <info>
-    <title>Ethnobiology and Conservation</title>
-    <title-short>Ethnobiol Conserv</title-short>
-    <id>http://www.zotero.org/styles/ethnobiology-and-conservation</id>
-    <link href="http://www.zotero.org/styles/ethnobiology-and-conservation" rel="self"/>
+    <title>Ethnobiology and Conservation 1th Edition</title>
+    <id>http://www.zotero.org/styles/ethnobiology-and-conservation-1th-edition</id>
+    <link href="http://www.zotero.org/styles/ethnobiology-and-conservation-1th-edition" rel="self"/>
     <link href="http://www.ethnobioconservation.com/index.php/ebc/pages/view/guidelines" rel="documentation"/>
+    <author>
```

Comment:
As your are adding to an already existing style, please make yourself a contributor after the other author/contributors.